api/internal/infrastructure/mongo/job.go

```
@@ -119,13 +153,40 @@
 	return c.Result[0], nil
 }
 
-func (r *Job) paginate(ctx context.Context, filter bson.M, pagination *usecasex.Pagination) ([]*job.Job, *usecasex.PageInfo, error) {
+func (r *Job) paginate(ctx context.Context, filter bson.M, pagination *interfaces.PaginationParam) ([]*job.Job, *usecasex.PageInfo, error) {
```

golangci-lint (1.62.2)

func `(*Job).paginate` is unused

(unused)